src/NUnitTestAdapter/EngineWrapper.cs

```diff
@@ -4,15 +4,13 @@
 
 using NUnit.Engine;
 using System;
-using System.Xml;
 
 namespace NUnit.VisualStudio.TestAdapter
 {
-    [Serializable]
-    internal class EngineWrapper : MarshalByRefObject, ITestEngine
+    public class EngineWrapper : MarshalByRefObject 
     {
-        [NonSerialized]
-        private ITestEngine _engine = new TestEngine();
+      
+        private readonly ITestEngine _engine = new TestEngine();
 
         public InternalTraceLevel InternalTraceLevel
```

Comment:
Since we are not implementing the interface anymore, I don't think many of these methods/properties are used anymore and can also be removed. Let's not worry about it now though, I think it is more important to get a fix out. Just a note for the next release.